compiler/hermes/include/hermes/core/Message.h

```diff
@@ -57,11 +58,17 @@ class Message final
   Message() = default;
 
 public:
-  void text(std::unique_ptr<MessageText> &&text) { _text = std::move(text); }
+  void text(std::unique_ptr<MessageText> &&text, SeverityCategory severity = INFO)
```

Comment:
Please remove `= INFO` , as I wrote before in another PR, we don't use implict default parameters